docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
You probably want to use the same set up as https://foss-eda-tools.googlesource.com/skywater-pdk-docs/+/refs/heads/master/docs/ (which uses `make-env` and similar).

Reply:
Can you provide more background as to why? This is working fine, what will we gain reworking it?

Reply:
(a bit of background from my side: we'd like to jump to sphinx ASAP so that we don't have to maintain two guides... we can then do such reworks no problem)

Reply:
@mgielda - It should be pretty easy to use that setup and it'll mean we don't have yet another migrate task which we put off for way to long?

Reply:
ok we can do it now but just not sure what you want here. That makefile is quite complex and introduces a conda dependency which is not mandatory to build some sphinx docs? which part of that Makefile do you care about?

Reply:
`make env` should support not using conda?